.buildkite/longruns/pipeline.yml

```diff
@@ -92,6 +92,19 @@ steps:
         artifact_paths: "longrun_hs_rhoe_third/*"
         agents:
           slurm_cpus_per_task: 8
+      
+      - label: ":computer: held suarez (ρe_tot) third order topography"
+        command: "julia --color=yes --project=examples --threads=8 examples/hybrid/driver.jl --forcing held_suarez --upwinding third_order --dt 300secs --t_end 1200days --fps 30 --job_id longrun_hs_rhoe_third --dt_save_to_sol 1days --dt_save_to_disk 10days --topography DCMIP200"
+        artifact_paths: "longrun_hs_rhoe_third/*"
+        agents:
+          slurm_cpus_per_task: 8
+      
+      - label: ":computer: held suarez (ρe_tot) third order topography"
+        command: "julia --color=yes --project=examples --threads=8 examples/hybrid/driver.jl --forcing held_suarez --upwinding third_order --dt 300secs --t_end 1200days --fps 30 --job_id longrun_hs_rhoe_third --dt_save_to_sol 1days --dt_save_to_disk 10days --topography DCMIP200"
+        artifact_paths: "longrun_hs_rhoe_third/*"
+        agents:
+          slurm_cpus_per_task: 8
```

Comment:
THe same simulation is added twice.

Reply:
Ah, and for longrun, I think we will also need sponge?

Reply:
cc @akshaysridhar